FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +50,15 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImageMapper.map(data, from: response))
+			case .failure(_):
```

Comment:
`(_)` is not needed, please remove it:
```suggestion
			case .failure:
```

Reply:
@danillahtin I've changed these two things base on your instructions. Thank you.